### PR TITLE
task 6 completed

### PR DESCRIPTION
**Pull request by jiancodemedia**
- added new model function checkArticleExist
- added new function getCommentsByArticleId
- added new test for get comments by article ID

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -125,4 +125,49 @@ describe('GET /api/articles', () => {
...
\ No newline at end of file
+         
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200: responds with an array of comments with correct properties', () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
+        const {comments} = body
+        comments.forEach((comment) => {
```
Comment (nickkagame) [inline __tests__/api.test.js:136]: we need a length check before the forEach.

File: __tests__/api.test.js [hunk 2]
```diff
@@ -125,4 +125,49 @@ describe('GET /api/articles', () => {
...
+    })
+  });
+  test('400: responds with an error message when invalid ID', () => {
+    return request(app)
+    .get('/api/articles/banana')
+    .expect(400)
+    .then ((res) => {
+      expect(res.body.msg).toBe('Invalid ID')
+    })
+  });
```
Comment (nickkagame) [inline __tests__/api.test.js:172]: we need one more test - Status 200, valid ID, but has no comments responds with an empty array of comments

**Review by nickkagame — COMMENTED** [review]
nice work Jian! just one more test needed.  if it passes you have all the behaviours you need!  See comments for details.
